UnitsNet.Tests/CustomCode/LengthTests.cs

```diff
@@ -32,6 +33,8 @@ public class LengthTests : LengthTestsBase
 
         protected override double FeetInOneMeter => 3.28084;
 
+        protected override double USSurveyFeetInOneMeter => 3937.0/1200.0;
```

Comment:
I'm not sure if this point comes across in the wiki guide, but it is kind of important that the test values are not functions or calculations, but values confirmed by some other means - to double check that the conversion function is in fact correct. My go to solution is to copy and paste the property name into google, and it usually comes up with the answer directly or points me to some online conversion tool. This value should be a number with a sufficient amount of digits.